pg_temp/pg_temp.py

```diff
@@ -358,6 +359,8 @@ def cleanup(self):
                 stdout=subprocess.DEVNULL,
             )
         elif self.pg_process:
+            pg_stop_cmd = ["pg_ctl", 'stop', '-D', self.pg_data_dir, '-m' 'fast']
```

Comment:
tiny sug: `:s/"/'/g` to keep the quotes consistent with the rest of the line / file. Using `pg_ctl` looks like a good idea to me at least :smile_cat:

Reply:
oops will update!